prac_05/wimbledon.py

```diff
@@ -0,0 +1,53 @@
+"""
+Emma Naumann
+CP1404 Practical 5
+Estimate: 30 minutes
+Actual:
+"""
+
+FILENAME = "wimbledon.csv"
+INDEX_COUNTRY = 1
+INDEX_CHAMPION = 2
+
+
+def main():
+    """Read data file and print details about Wimbledon champions and countries."""
+    records = get_records(FILENAME)
+    champion_to_count, countries = process_records(records)
+    display_results(champion_to_count, countries)
+
+
+def get_records(filename):
+    """Get records from file and store as a list of lists."""
+    records = []
+    with open(FILENAME, "r", encoding="utf-8-sig") as in_file:
+        in_file.readline()
+        for line in in_file:
+            parts = line.strip().split(",")
```

Comment:
I don't know if .strip() is necessary for the desired outcome to work, I would recommend giving it a go without it :)